packages/jest-config/src/normalize.js

```diff
@@ -436,10 +435,10 @@ function normalize(config, argv) {
     return newConfig;
   }, newConfig);
 
-  // If argv.json is set the coverageReporters array must be overridden
+  // If argv.json is set the coverageReporters to json so we don't polute STDOUT
```

Comment:
Please use: "If argv.json is set, coverageReporters shouldn't print a text report."